external-docs/mkdocs-static-site/docs/about.md

```diff
@@ -0,0 +1,38 @@
+# Qui Tres divitior
```

Comment:
Let's not commit any of the files in `mkdocs-static-site` since they can be auto-generated on Jenkins.

Reply:
Well, except for the markdown files of course! But can you move them out of the `mkdocs-static-site` directory into `external-docs`? There's no point having a directory which only contains a directory.